compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -447,6 +448,8 @@ class Definitions {
     def Long_* = Long_mulR.symbol
     lazy val Long_divR   = LongClass.requiredMethodRef(nme.DIV, List(LongType))
     def Long_/ = Long_divR.symbol
+  val CommutativePrimitiveOperations = new PerRun[collection.Set[Symbol]](implicit ctx =>
```

Comment:
Don't put stuff inside Definitions, unless its frequently used by multiple phases.

Same applies for SeqFactoryType.